solutions/org-policies/setters.yaml

```diff
@@ -32,6 +32,7 @@ data:
   # a list of allowed trusted image projects, see YAML file for more info:
   # organization/compute-trusted-image-projects.yaml
   # this setting can be left as default or modified as required
+  # fortigcp-project-001 and windows-cloud are required for the perimeter project
```

Comment:
should we add them to the `allowed-trusted-image-projects` list by default?

Reply:
defining the list of trusted image at an organization level should be a separate story.